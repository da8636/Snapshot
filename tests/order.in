SET a 5 7 4
GET a
SORT a
GET a
BYE 
