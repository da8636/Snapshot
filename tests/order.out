ok

[5 7 4]

ok

[4 5 7]

bye
